src/app.js

```diff
@@ -8,10 +8,10 @@ const Importer = require('./importer');
 const express = require('express');
 const cookieParser = require('cookie-parser');
 
-console.log(config.name);
+console.log(config.name); // eslint-disable-line
```

Comment:
It's not sage to suppress errors this way. It's better to specify exect rule such as `// eslint-disable-line no-console`. In that case it will suppress just one rule, but check everything else. With your solution, if for example config would be undefined, it won't show you any error, but with my approch it will, but it won't complain about `console.log` usage itself.